modules/ROOT/pages/configure-client-management-entra-id.adoc

```diff
@@ -67,11 +67,11 @@ When https://docs.microsoft.com/en-us/graph/api/application-post-applications?vi
 [[client-secrets]]
 === Client Secrets
 
-Client secrets (application password) use a string value in the Microsoft Entra application to authenticate instead of a certificate for identity. 
+Client secrets (application password) use a string value in the Microsoft Entra application for authentication instead of a certificate for identity. 
 
-Client secrets are set to expire six months from the date of creation per Microsoft's security recommendations. 
+Client secrets are set to expire six months from the date of creation per Microsoft's security recommendations and can't be reset in the Access Management UI.  
```

Comment:
I got confused by `can't be reset in the access management ui`. Sounds like you can't reset secrets but effectively you can. Do you mean reset the expiration time?